files/primary.conf

```diff
@@ -313,6 +313,13 @@ verb 3
 # can automatically reconnect.
 explicit-exit-notify 1
 
+# Store the certificates the clients uses upon connection to this directory.
```

Comment:
"clients uses" is _poor_ English.

Reply:
My bad.  Copied from man page without proofreading it.